src/code_1AF230.c

```diff
@@ -1,19 +1,33 @@
 #include "common.h"
 
-typedef struct StarPower { void* dmaStart; void* dmaEnd; void* dmaDest; Bytecode* init; } StarPower;
+typedef struct StarPowerEntry {
+    /* 0x04 */ void* dmaStart;
+    /* 0x08 */ void* dmaEnd;
+    /* 0x0C */ void* dmaDest;S
+    /* 0x10 */ Bytecode* init;
+} StarPowerEntry; // size = 0x14
 
-extern StarPower D_8029C7D0[13];
+extern StarPowerEntry D_8029C7D0[13];
 
-typedef struct WhateverThisIs { s8 starpoints; s8 a; s8 b; s8 c; s32 w; s32 x; s32 y; s32 z; } WhateverThisIs;
+typedef struct StarPowerDesc {
+    /* 0x00 */ s8 starPoints;
+    /* 0x01 */ s8 a;
+    /* 0x02 */ s8 b;
+    /* 0x03 */ s8 c;
+    /* 0x04 */ s32 w;
+    /* 0x08 */ s32 x;
+    /* 0x0C */ s32 y;
+    /* 0x10 */ s32 z;
```

Comment:
Maybe call these `unk_xx`? Doesn't really matter tbh

Reply:
Yeah, imo these should be unk_xx if we don't have a good name